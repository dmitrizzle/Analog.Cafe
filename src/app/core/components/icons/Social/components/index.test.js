import "jest-styled-components"

import { shallow } from "enzyme"
import React from "react"

import Facebook from "./Facebook"
import Instagram from "./Instagram"
import Twitter from "./Twitter"

it("Render Facebook, matches snapshot", () => {
  const element = shallow(<Facebook />)
  expect(element).toMatchSnapshot()
})

it("Render Instagram, matches snapshot", () => {
  const element = shallow(<Instagram />)
  expect(element).toMatchSnapshot()
})

it("Render Twitter, matches snapshot", () => {
  const element = shallow(<Twitter />)
  expect(element).toMatchSnapshot()
})
