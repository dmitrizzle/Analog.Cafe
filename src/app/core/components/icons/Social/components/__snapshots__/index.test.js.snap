// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Facebook, matches snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <path
        d="M178.4 258.3L178.4 158.5 211.9 158.5 216.9 119.6 178.4 119.6 178.4 94.8C178.4 83.5 181.5 75.8 197.7 75.8L218.3 75.8 218.3 41C214.7 40.6 202.5 39.5 188.3 39.5 158.6 39.5 138.2 57.6 138.2 90.9L138.2 119.6 104.7 119.6 104.7 158.5 138.2 158.5 138.2 258.3 178.4 258.3Z"
        style={
                Object {
                        "fill": "#FFFFFF",
                      }
        }
/>,
      "viewBox": "0 0 325 325",
      "xmlns": "http://www.w3.org/2000/svg",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "d": "M178.4 258.3L178.4 158.5 211.9 158.5 216.9 119.6 178.4 119.6 178.4 94.8C178.4 83.5 181.5 75.8 197.7 75.8L218.3 75.8 218.3 41C214.7 40.6 202.5 39.5 188.3 39.5 158.6 39.5 138.2 57.6 138.2 90.9L138.2 119.6 104.7 119.6 104.7 158.5 138.2 158.5 138.2 258.3 178.4 258.3Z",
        "style": Object {
          "fill": "#FFFFFF",
        },
      },
      "ref": null,
      "rendered": null,
      "type": "path",
    },
    "type": "svg",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <path
          d="M178.4 258.3L178.4 158.5 211.9 158.5 216.9 119.6 178.4 119.6 178.4 94.8C178.4 83.5 181.5 75.8 197.7 75.8L218.3 75.8 218.3 41C214.7 40.6 202.5 39.5 188.3 39.5 158.6 39.5 138.2 57.6 138.2 90.9L138.2 119.6 104.7 119.6 104.7 158.5 138.2 158.5 138.2 258.3 178.4 258.3Z"
          style={
                    Object {
                              "fill": "#FFFFFF",
                            }
          }
/>,
        "viewBox": "0 0 325 325",
        "xmlns": "http://www.w3.org/2000/svg",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "d": "M178.4 258.3L178.4 158.5 211.9 158.5 216.9 119.6 178.4 119.6 178.4 94.8C178.4 83.5 181.5 75.8 197.7 75.8L218.3 75.8 218.3 41C214.7 40.6 202.5 39.5 188.3 39.5 158.6 39.5 138.2 57.6 138.2 90.9L138.2 119.6 104.7 119.6 104.7 158.5 138.2 158.5 138.2 258.3 178.4 258.3Z",
          "style": Object {
            "fill": "#FFFFFF",
          },
        },
        "ref": null,
        "rendered": null,
        "type": "path",
      },
      "type": "svg",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;

exports[`Render Instagram, matches snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <path
        d="M400 72.1c106.8 0 119.5 0.4 161.6 2.3 39 1.8 60.2 8.3 74.3 13.8 18.7 7.3 32 15.9 46 29.9 14 14 22.7 27.3 29.9 46 5.5 14.1 12 35.3 13.8 74.3 1.9 42.2 2.3 54.8 2.3 161.6s-0.4 119.5-2.3 161.6c-1.8 39-8.3 60.2-13.8 74.3 -7.3 18.7-15.9 32-29.9 46 -14 14-27.3 22.7-46 29.9 -14.1 5.5-35.3 12-74.3 13.8 -42.2 1.9-54.8 2.3-161.6 2.3 -106.8 0-119.5-0.4-161.6-2.3 -39-1.8-60.2-8.3-74.3-13.8 -18.7-7.3-32-15.9-46-29.9 -14-14-22.7-27.3-29.9-46 -5.5-14.1-12-35.3-13.8-74.3 -1.9-42.2-2.3-54.8-2.3-161.6s0.4-119.5 2.3-161.6c1.8-39 8.3-60.2 13.8-74.3 7.3-18.7 15.9-32 29.9-46 14-14 27.3-22.7 46-29.9 14.1-5.5 35.3-12 74.3-13.8C280.5 72.5 293.2 72.1 400 72.1M400 0c-108.6 0-122.3 0.5-164.9 2.4 -42.6 1.9-71.7 8.7-97.1 18.6C111.7 31.2 89.4 44.9 67.1 67.1 44.9 89.4 31.2 111.7 21 138 11.1 163.4 4.4 192.5 2.4 235.1c-1.9 42.7-2.4 56.3-2.4 164.9 0 108.6 0.5 122.3 2.4 164.9 1.9 42.6 8.7 71.7 18.6 97.1 10.2 26.3 23.9 48.6 46.1 70.9 22.2 22.2 44.5 35.9 70.8 46.1 25.4 9.9 54.5 16.6 97.1 18.6 42.7 1.9 56.3 2.4 164.9 2.4 108.6 0 122.3-0.5 164.9-2.4 42.6-1.9 71.7-8.7 97.1-18.6 26.3-10.2 48.6-23.9 70.9-46.1 22.2-22.2 35.9-44.5 46.1-70.8 9.9-25.4 16.7-54.5 18.6-97.1 1.9-42.7 2.4-56.3 2.4-164.9 0-108.6-0.5-122.3-2.4-164.9 -1.9-42.6-8.7-71.7-18.6-97.1 -10.2-26.3-23.9-48.6-46.1-70.8 -22.2-22.2-44.5-35.9-70.9-46.1 -25.4-9.9-54.5-16.7-97.1-18.6C522.3 0.5 508.6 0 400 0M400 194.6c-113.4 0-205.4 92-205.4 205.4S286.6 605.4 400 605.4c113.4 0 205.4-92 205.4-205.4S513.4 194.6 400 194.6M400 533.3c-73.6 0-133.3-59.7-133.3-133.3s59.7-133.3 133.3-133.3 133.3 59.7 133.3 133.3S473.6 533.3 400 533.3M661.5 186.5c0 26.5-21.5 48-48 48 -26.5 0-48-21.5-48-48 0-26.5 21.5-48 48-48C640 138.5 661.5 160 661.5 186.5"
        style={
                Object {
                        "fill": "#FFFFFF",
                      }
        }
/>,
      "viewBox": "0 0 800 800",
      "xmlns": "http://www.w3.org/2000/svg",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "d": "M400 72.1c106.8 0 119.5 0.4 161.6 2.3 39 1.8 60.2 8.3 74.3 13.8 18.7 7.3 32 15.9 46 29.9 14 14 22.7 27.3 29.9 46 5.5 14.1 12 35.3 13.8 74.3 1.9 42.2 2.3 54.8 2.3 161.6s-0.4 119.5-2.3 161.6c-1.8 39-8.3 60.2-13.8 74.3 -7.3 18.7-15.9 32-29.9 46 -14 14-27.3 22.7-46 29.9 -14.1 5.5-35.3 12-74.3 13.8 -42.2 1.9-54.8 2.3-161.6 2.3 -106.8 0-119.5-0.4-161.6-2.3 -39-1.8-60.2-8.3-74.3-13.8 -18.7-7.3-32-15.9-46-29.9 -14-14-22.7-27.3-29.9-46 -5.5-14.1-12-35.3-13.8-74.3 -1.9-42.2-2.3-54.8-2.3-161.6s0.4-119.5 2.3-161.6c1.8-39 8.3-60.2 13.8-74.3 7.3-18.7 15.9-32 29.9-46 14-14 27.3-22.7 46-29.9 14.1-5.5 35.3-12 74.3-13.8C280.5 72.5 293.2 72.1 400 72.1M400 0c-108.6 0-122.3 0.5-164.9 2.4 -42.6 1.9-71.7 8.7-97.1 18.6C111.7 31.2 89.4 44.9 67.1 67.1 44.9 89.4 31.2 111.7 21 138 11.1 163.4 4.4 192.5 2.4 235.1c-1.9 42.7-2.4 56.3-2.4 164.9 0 108.6 0.5 122.3 2.4 164.9 1.9 42.6 8.7 71.7 18.6 97.1 10.2 26.3 23.9 48.6 46.1 70.9 22.2 22.2 44.5 35.9 70.8 46.1 25.4 9.9 54.5 16.6 97.1 18.6 42.7 1.9 56.3 2.4 164.9 2.4 108.6 0 122.3-0.5 164.9-2.4 42.6-1.9 71.7-8.7 97.1-18.6 26.3-10.2 48.6-23.9 70.9-46.1 22.2-22.2 35.9-44.5 46.1-70.8 9.9-25.4 16.7-54.5 18.6-97.1 1.9-42.7 2.4-56.3 2.4-164.9 0-108.6-0.5-122.3-2.4-164.9 -1.9-42.6-8.7-71.7-18.6-97.1 -10.2-26.3-23.9-48.6-46.1-70.8 -22.2-22.2-44.5-35.9-70.9-46.1 -25.4-9.9-54.5-16.7-97.1-18.6C522.3 0.5 508.6 0 400 0M400 194.6c-113.4 0-205.4 92-205.4 205.4S286.6 605.4 400 605.4c113.4 0 205.4-92 205.4-205.4S513.4 194.6 400 194.6M400 533.3c-73.6 0-133.3-59.7-133.3-133.3s59.7-133.3 133.3-133.3 133.3 59.7 133.3 133.3S473.6 533.3 400 533.3M661.5 186.5c0 26.5-21.5 48-48 48 -26.5 0-48-21.5-48-48 0-26.5 21.5-48 48-48C640 138.5 661.5 160 661.5 186.5",
        "style": Object {
          "fill": "#FFFFFF",
        },
      },
      "ref": null,
      "rendered": null,
      "type": "path",
    },
    "type": "svg",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <path
          d="M400 72.1c106.8 0 119.5 0.4 161.6 2.3 39 1.8 60.2 8.3 74.3 13.8 18.7 7.3 32 15.9 46 29.9 14 14 22.7 27.3 29.9 46 5.5 14.1 12 35.3 13.8 74.3 1.9 42.2 2.3 54.8 2.3 161.6s-0.4 119.5-2.3 161.6c-1.8 39-8.3 60.2-13.8 74.3 -7.3 18.7-15.9 32-29.9 46 -14 14-27.3 22.7-46 29.9 -14.1 5.5-35.3 12-74.3 13.8 -42.2 1.9-54.8 2.3-161.6 2.3 -106.8 0-119.5-0.4-161.6-2.3 -39-1.8-60.2-8.3-74.3-13.8 -18.7-7.3-32-15.9-46-29.9 -14-14-22.7-27.3-29.9-46 -5.5-14.1-12-35.3-13.8-74.3 -1.9-42.2-2.3-54.8-2.3-161.6s0.4-119.5 2.3-161.6c1.8-39 8.3-60.2 13.8-74.3 7.3-18.7 15.9-32 29.9-46 14-14 27.3-22.7 46-29.9 14.1-5.5 35.3-12 74.3-13.8C280.5 72.5 293.2 72.1 400 72.1M400 0c-108.6 0-122.3 0.5-164.9 2.4 -42.6 1.9-71.7 8.7-97.1 18.6C111.7 31.2 89.4 44.9 67.1 67.1 44.9 89.4 31.2 111.7 21 138 11.1 163.4 4.4 192.5 2.4 235.1c-1.9 42.7-2.4 56.3-2.4 164.9 0 108.6 0.5 122.3 2.4 164.9 1.9 42.6 8.7 71.7 18.6 97.1 10.2 26.3 23.9 48.6 46.1 70.9 22.2 22.2 44.5 35.9 70.8 46.1 25.4 9.9 54.5 16.6 97.1 18.6 42.7 1.9 56.3 2.4 164.9 2.4 108.6 0 122.3-0.5 164.9-2.4 42.6-1.9 71.7-8.7 97.1-18.6 26.3-10.2 48.6-23.9 70.9-46.1 22.2-22.2 35.9-44.5 46.1-70.8 9.9-25.4 16.7-54.5 18.6-97.1 1.9-42.7 2.4-56.3 2.4-164.9 0-108.6-0.5-122.3-2.4-164.9 -1.9-42.6-8.7-71.7-18.6-97.1 -10.2-26.3-23.9-48.6-46.1-70.8 -22.2-22.2-44.5-35.9-70.9-46.1 -25.4-9.9-54.5-16.7-97.1-18.6C522.3 0.5 508.6 0 400 0M400 194.6c-113.4 0-205.4 92-205.4 205.4S286.6 605.4 400 605.4c113.4 0 205.4-92 205.4-205.4S513.4 194.6 400 194.6M400 533.3c-73.6 0-133.3-59.7-133.3-133.3s59.7-133.3 133.3-133.3 133.3 59.7 133.3 133.3S473.6 533.3 400 533.3M661.5 186.5c0 26.5-21.5 48-48 48 -26.5 0-48-21.5-48-48 0-26.5 21.5-48 48-48C640 138.5 661.5 160 661.5 186.5"
          style={
                    Object {
                              "fill": "#FFFFFF",
                            }
          }
/>,
        "viewBox": "0 0 800 800",
        "xmlns": "http://www.w3.org/2000/svg",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "d": "M400 72.1c106.8 0 119.5 0.4 161.6 2.3 39 1.8 60.2 8.3 74.3 13.8 18.7 7.3 32 15.9 46 29.9 14 14 22.7 27.3 29.9 46 5.5 14.1 12 35.3 13.8 74.3 1.9 42.2 2.3 54.8 2.3 161.6s-0.4 119.5-2.3 161.6c-1.8 39-8.3 60.2-13.8 74.3 -7.3 18.7-15.9 32-29.9 46 -14 14-27.3 22.7-46 29.9 -14.1 5.5-35.3 12-74.3 13.8 -42.2 1.9-54.8 2.3-161.6 2.3 -106.8 0-119.5-0.4-161.6-2.3 -39-1.8-60.2-8.3-74.3-13.8 -18.7-7.3-32-15.9-46-29.9 -14-14-22.7-27.3-29.9-46 -5.5-14.1-12-35.3-13.8-74.3 -1.9-42.2-2.3-54.8-2.3-161.6s0.4-119.5 2.3-161.6c1.8-39 8.3-60.2 13.8-74.3 7.3-18.7 15.9-32 29.9-46 14-14 27.3-22.7 46-29.9 14.1-5.5 35.3-12 74.3-13.8C280.5 72.5 293.2 72.1 400 72.1M400 0c-108.6 0-122.3 0.5-164.9 2.4 -42.6 1.9-71.7 8.7-97.1 18.6C111.7 31.2 89.4 44.9 67.1 67.1 44.9 89.4 31.2 111.7 21 138 11.1 163.4 4.4 192.5 2.4 235.1c-1.9 42.7-2.4 56.3-2.4 164.9 0 108.6 0.5 122.3 2.4 164.9 1.9 42.6 8.7 71.7 18.6 97.1 10.2 26.3 23.9 48.6 46.1 70.9 22.2 22.2 44.5 35.9 70.8 46.1 25.4 9.9 54.5 16.6 97.1 18.6 42.7 1.9 56.3 2.4 164.9 2.4 108.6 0 122.3-0.5 164.9-2.4 42.6-1.9 71.7-8.7 97.1-18.6 26.3-10.2 48.6-23.9 70.9-46.1 22.2-22.2 35.9-44.5 46.1-70.8 9.9-25.4 16.7-54.5 18.6-97.1 1.9-42.7 2.4-56.3 2.4-164.9 0-108.6-0.5-122.3-2.4-164.9 -1.9-42.6-8.7-71.7-18.6-97.1 -10.2-26.3-23.9-48.6-46.1-70.8 -22.2-22.2-44.5-35.9-70.9-46.1 -25.4-9.9-54.5-16.7-97.1-18.6C522.3 0.5 508.6 0 400 0M400 194.6c-113.4 0-205.4 92-205.4 205.4S286.6 605.4 400 605.4c113.4 0 205.4-92 205.4-205.4S513.4 194.6 400 194.6M400 533.3c-73.6 0-133.3-59.7-133.3-133.3s59.7-133.3 133.3-133.3 133.3 59.7 133.3 133.3S473.6 533.3 400 533.3M661.5 186.5c0 26.5-21.5 48-48 48 -26.5 0-48-21.5-48-48 0-26.5 21.5-48 48-48C640 138.5 661.5 160 661.5 186.5",
          "style": Object {
            "fill": "#FFFFFF",
          },
        },
        "ref": null,
        "rendered": null,
        "type": "path",
      },
      "type": "svg",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;

exports[`Render Twitter, matches snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <path
        d="M153.6 301.6c94.3 0 145.9-78.2 145.9-145.9 0-2.2 0-4.4-0.1-6.6 10-7.2 18.7-16.3 25.6-26.6 -9.2 4.1-19.1 6.8-29.5 8.1 10.6-6.3 18.7-16.4 22.6-28.4 -9.9 5.9-20.9 10.1-32.6 12.4 -9.4-10-22.7-16.2-37.4-16.2 -28.3 0-51.3 23-51.3 51.3 0 4 0.5 7.9 1.3 11.7 -42.6-2.1-80.4-22.6-105.7-53.6 -4.4 7.6-6.9 16.4-6.9 25.8 0 17.8 9.1 33.5 22.8 42.7 -8.4-0.3-16.3-2.6-23.2-6.4 0 0.2 0 0.4 0 0.7 0 24.8 17.7 45.6 41.1 50.3 -4.3 1.2-8.8 1.8-13.5 1.8 -3.3 0-6.5-0.3-9.6-0.9 6.5 20.4 25.5 35.2 47.9 35.6 -17.6 13.8-39.7 22-63.7 22 -4.1 0-8.2-0.2-12.2-0.7C97.7 293.1 124.7 301.6 153.6 301.6"
        style={
                Object {
                        "fill": "#FFFFFF",
                      }
        }
/>,
      "viewBox": "0 0 400 400",
      "xmlns": "http://www.w3.org/2000/svg",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "d": "M153.6 301.6c94.3 0 145.9-78.2 145.9-145.9 0-2.2 0-4.4-0.1-6.6 10-7.2 18.7-16.3 25.6-26.6 -9.2 4.1-19.1 6.8-29.5 8.1 10.6-6.3 18.7-16.4 22.6-28.4 -9.9 5.9-20.9 10.1-32.6 12.4 -9.4-10-22.7-16.2-37.4-16.2 -28.3 0-51.3 23-51.3 51.3 0 4 0.5 7.9 1.3 11.7 -42.6-2.1-80.4-22.6-105.7-53.6 -4.4 7.6-6.9 16.4-6.9 25.8 0 17.8 9.1 33.5 22.8 42.7 -8.4-0.3-16.3-2.6-23.2-6.4 0 0.2 0 0.4 0 0.7 0 24.8 17.7 45.6 41.1 50.3 -4.3 1.2-8.8 1.8-13.5 1.8 -3.3 0-6.5-0.3-9.6-0.9 6.5 20.4 25.5 35.2 47.9 35.6 -17.6 13.8-39.7 22-63.7 22 -4.1 0-8.2-0.2-12.2-0.7C97.7 293.1 124.7 301.6 153.6 301.6",
        "style": Object {
          "fill": "#FFFFFF",
        },
      },
      "ref": null,
      "rendered": null,
      "type": "path",
    },
    "type": "svg",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <path
          d="M153.6 301.6c94.3 0 145.9-78.2 145.9-145.9 0-2.2 0-4.4-0.1-6.6 10-7.2 18.7-16.3 25.6-26.6 -9.2 4.1-19.1 6.8-29.5 8.1 10.6-6.3 18.7-16.4 22.6-28.4 -9.9 5.9-20.9 10.1-32.6 12.4 -9.4-10-22.7-16.2-37.4-16.2 -28.3 0-51.3 23-51.3 51.3 0 4 0.5 7.9 1.3 11.7 -42.6-2.1-80.4-22.6-105.7-53.6 -4.4 7.6-6.9 16.4-6.9 25.8 0 17.8 9.1 33.5 22.8 42.7 -8.4-0.3-16.3-2.6-23.2-6.4 0 0.2 0 0.4 0 0.7 0 24.8 17.7 45.6 41.1 50.3 -4.3 1.2-8.8 1.8-13.5 1.8 -3.3 0-6.5-0.3-9.6-0.9 6.5 20.4 25.5 35.2 47.9 35.6 -17.6 13.8-39.7 22-63.7 22 -4.1 0-8.2-0.2-12.2-0.7C97.7 293.1 124.7 301.6 153.6 301.6"
          style={
                    Object {
                              "fill": "#FFFFFF",
                            }
          }
/>,
        "viewBox": "0 0 400 400",
        "xmlns": "http://www.w3.org/2000/svg",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "d": "M153.6 301.6c94.3 0 145.9-78.2 145.9-145.9 0-2.2 0-4.4-0.1-6.6 10-7.2 18.7-16.3 25.6-26.6 -9.2 4.1-19.1 6.8-29.5 8.1 10.6-6.3 18.7-16.4 22.6-28.4 -9.9 5.9-20.9 10.1-32.6 12.4 -9.4-10-22.7-16.2-37.4-16.2 -28.3 0-51.3 23-51.3 51.3 0 4 0.5 7.9 1.3 11.7 -42.6-2.1-80.4-22.6-105.7-53.6 -4.4 7.6-6.9 16.4-6.9 25.8 0 17.8 9.1 33.5 22.8 42.7 -8.4-0.3-16.3-2.6-23.2-6.4 0 0.2 0 0.4 0 0.7 0 24.8 17.7 45.6 41.1 50.3 -4.3 1.2-8.8 1.8-13.5 1.8 -3.3 0-6.5-0.3-9.6-0.9 6.5 20.4 25.5 35.2 47.9 35.6 -17.6 13.8-39.7 22-63.7 22 -4.1 0-8.2-0.2-12.2-0.7C97.7 293.1 124.7 301.6 153.6 301.6",
          "style": Object {
            "fill": "#FFFFFF",
          },
        },
        "ref": null,
        "rendered": null,
        "type": "path",
      },
      "type": "svg",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
